### PR TITLE
boiler plate juju info cmd

### PR DESCRIPTION
**Pull request by hmlanigan**
## Description of change

Add feature flag for all Charm Hub integration work: "charm-hub"
Add basic code for the new ```juju info``` command part of Charm Hub integration work.  Other pieces such has charm validation and the api need to be worked out.

## QA steps
```console
$ export JUJU_DEV_FEATURE_FLAGS=charm-hub
$ juju bootstrap localhost --build-agent
$ juju info
ERROR expected a charm or bundle name
$ juju info test
ERROR business logic of info command not implemented
```

## Documentation changes

Will be needed for the charm hub integration work for juju 2.9

### CODE REVIEW

File: cmd/juju/charmhub/info.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	"github.com/juju/juju/api/charmhub"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+const (
+	infoSummary = "Displays detailed information about charm hub charms."
+	infoDoc     = `
+The charm can be specified by name or by path. Names are looked for both in the
+store and in the installed charms.
```
Comment (manadart) [inline cmd/juju/charmhub/info.go:20]: "deployed charms"?

Comment (manadart) [inline cmd/juju/charmhub/info.go:20]: "deployed charms"?

**Comment by petevg** [comment]
$$merge$$

**Comment by hmlanigan** [comment]
$$merge$$

**Pull request merged**